.github/workflows/ci.yml

```diff
@@ -65,3 +65,22 @@ jobs:
       run: make probe-locally
     - name: Bring down local DSS instance
       run: make down-locally
+    - name: Checkout monitoring repository
+      uses: actions/checkout@v2
+      with:
+        repository: interuss/monitoring
+        submodules: true
+        path: monitoring-repo
+        ref: main  # consider checking out a tag instead? Downside is that we'd need to update it with each release
+    - name: Run qualifier in monitoring repository
+      run: |
+        set -e
+        cd monitoring-repo
+        # Set the docker-compose file to use the locally built DSS instance
+        # Note that on OS X you'll need to use `sed -i ''` instead of `sed -i`
+        sed -i 's|interuss/dss:[^ ]*|interuss-local/dss:latest|g' build/dev/docker-compose.yaml
```

Comment:
The ability to bring up a local DSS instance should certainly exist in the DSS repo -- we shouldn't need to rely on the content of another repository to do that.

Since uss_qualifier is designed to test systems however they're deployed, we should tell it how our DSS system is deployed when we want to test it.  That means we should have a test configuration for testing the DSS tracked in this repo.  The configuration will probably look a lot like [this one](https://github.com/interuss/monitoring/blob/main/monitoring/uss_qualifier/configurations/dev/dss_probing.yaml), but we should remove the $refs because the targets exist in another repo and we don't want to be dependent on them for the configuration to be valid.  To do this, we can run uss_qualifier with [`--config-output`](https://github.com/interuss/monitoring/blob/8b760a6bb16d0f7fc90ed278ffd167958b5cb409/monitoring/uss_qualifier/main.py#L47).

Once we have a locally-deployed system to test and a configuration specifying how to test it, we should be able to just run uss_qualifier directly similar to how we [run `prober` directly](https://github.com/interuss/dss/blob/73c267551e3a047acb1fd53fc90c7d97efeba814/build/dev/probe_locally.sh#L38).  `monitoring`'s run_locally.sh is probably a good hint on what [that command to run uss_qualifier](https://github.com/interuss/monitoring/blob/8b760a6bb16d0f7fc90ed278ffd167958b5cb409/monitoring/uss_qualifier/run_locally.sh#L63) should look like, but we shouldn't need to run the script in the other repo since that's just an example and not [the generally-expected usage of uss-qualifier](https://github.com/interuss/monitoring/tree/main/monitoring/uss_qualifier#usage).

It is probably also a good idea to (separately) modernize [the local DSS deployment approach in this repo](https://github.com/interuss/dss/blob/master/build/dev/docker-compose_dss.yaml) to look more like the one in the monitoring repo, but the two repos should not be directly coupled.